experimental/routes/demo_app_routes.dart

```diff
@@ -5,10 +5,18 @@ abstract class DemoRoutes {
   DemoRoutes._();
   static const demoHome = _DemoPaths.demoHome;
   static const demoLogin = _DemoPaths.demoLogin;
+  static const demoProfile = _DemoPaths.demoHome + _DemoPaths.demoProfile;
+  static const demoUpdate = _DemoPaths.demoHome + _DemoPaths.demoUpdate;
+  static const demoDelete = _DemoPaths.demoHome + _DemoPaths.demoDelete;
+  static const demoCreate = _DemoPaths.demoHome + _DemoPaths.demoCreate;
 }
 
 abstract class _DemoPaths {
   _DemoPaths._();
   static const demoHome = '/demoHome';
   static const demoLogin = '/demologin';
+  static const demoProfile = '/demoProfile';
+  static const demoCreate = '/demoCreate';
+  static const demoUpdate = '/demoUpdate';
+  static const demoDelete = '/demoDelete';
```

Comment:
ご対応ありがとうございます！
細かいところになりますが、`Profile`も入れた方がわかりやすいと思うので、下記のように修正お願いします。
```suggestion
  static const demoCreate = '/demoProfileCreate';
  static const demoUpdate = '/demoProfileUpdate';
  static const demoDelete = '/demoProfileDelete';
```